src/systemTest/java/com/flightstats/hub/clients/hub/channel/ChannelItemResourceClient.java

```diff
@@ -43,6 +43,14 @@
                                               @Path("s") int second,
                                               @Query("location") Location location);
 
+    @GET("/channel/{channelName}/{Y}/{M}/{D}/{h}")
+    Call<TimeQueryResult> getItemsHourPath(@Path("channelName") String channelName,
+                                              @Path("Y") int year,
```

Comment:
Nit, though I guess it agreement with the other time path endpoints. Why abbreviate Y, M, D, etc here, but use full names on other end points (like directionalItems). And then, why lowercase "h", but upper case the others? I thought maybe it was because it was the abbreviations used by DateTimeFormatter, but that doesn't match up either.

Like I said, it's a nit, maybe not worth noting, just struck me as odd/inconsistent.

Reply:
The directional items thing is my fault. I got tired of parsing out an existing content key back into args just to construct 'em back into the same format content key as it started out.